identity_storage/src/key_storage/memstore.rs

```diff
@@ -65,7 +70,8 @@ impl JwkStorage for JwkMemStore {
 
     let mut jwk: Jwk = ed25519::encode_jwk(&private_key, &public_key);
     jwk.set_alg(alg.name());
-    let public_jwk: Jwk = jwk.to_public();
+    let mut public_jwk: Jwk = jwk.to_public().unwrap();
```

Comment:
`Unwrap` is OK because the `None` variant only occurs for `kty = oct`.

Reply:
Note to self: Comment this directly in code.